### PR TITLE
change the seccompProfile field description to match the imlpementation

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
#### What type of PR is this?

/kind cleanup
/sig node

see: https://github.com/kubernetes/kubernetes/pull/117020#discussion_r1157263306

#### What this PR does / why we need it:

Random cleanup of a doc after the code change was merged

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/api/core/v1/types.go [hunk 1]
```diff
@@ -3838,7 +3838,7 @@ type SeccompProfile struct {
 	// localhostProfile indicates a profile defined in a file on the node should be used.
 	// The profile must be preconfigured on the node to work.
 	// Must be a descending path, relative to the kubelet's configured seccomp profile location.
-	// Must only be set if type is "Localhost".
+	// Must be set if type is "Localhost".
```
[the remark this answers is not in the document]
Comment (SergeyKanzhelev) [reply staging/src/k8s.io/api/core/v1/types.go:3841]: Yes: https://github.com/kubernetes/kubernetes/blob/e9e60316d303a17c81ed856d6c82a4823b4327f8/pkg/apis/core/validation/validation.go#L4262-L4272

Will change to explain both shortly

**Review by mrunalp — APPROVED** [review]

**Pull request merged**